### PR TITLE
Functional Google Maps autocomplete

### LINKED ISSUE #2: Basic address entry
Add the address text input field with Google autocomplete (there's more than likely a React component for it somewhere)

As a user, I want to enter my home address into the application so that I can determine the food security near my home.

Scenario A: The user wants to enter their address
Given: the user is on the application's main page
When: the user enters their address
Then: the application displays their food security status

### PR DESCRIPTION
**Pull request by evan10s**
Changes
----

- Got Google Maps autocomplete functionality working on a basic level

In #76, we will actually link this autocomplete with the rest of the UI, but for now the component will return the lat/long for the location selected, even if that isn't displayed in the UI.  The other UI complexities as part of #2 will also be address with #76.

Screenshots
----

- 
image (8)

Resolved issues
-----

Closes #2

### CODE REVIEW

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>ℹ</td>
      <td>This pull request includes client dependency changes.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a5dc341ca2e9b32c462a9f442a0f598a79cc903f
</p>

File: client/src/components/MapContainer.js [hunk 1]
```diff
@@ -1,18 +1,44 @@
 import React from "react";
-import GoogleMap from "./map/GoogleMap";
 import Filters from "./Filters.js";
 import "./css/map.css";
+import AutocompleteV2 from "./map/AutocompleteV2";
+import {compose, withProps} from "recompose";
+import {GoogleMap, Marker, withGoogleMap, withScriptjs} from "react-google-maps"
```
Comment (hannahli23) [inline client/src/components/MapContainer.js]: Semicolon here!

File: client/src/components/MapContainer.js [hunk 2]
```diff
@@ -1,18 +1,44 @@
...
         );
     }
 }
 
+const MapContainer = compose(
+    withProps({
+        googleMapURL: `https://maps.googleapis.com/maps/api/js?key=${process.env.REACT_APP_GOOGLE_MAPS_API_KEY}&libraries=geometry,drawing,places`,
+        loadingElement: (<p>Loading</p>),
+        containerElement: <div
+            style={{minHeight: "400px", height: "500px", maxHeight: "100px", minWidth: '500px', maxWidth: "800px"}}/>,
```
Comment (hannahli23) [inline client/src/components/MapContainer.js]: your minWidth needs to be in double quotes

File: client/src/components/MapContainer.js [hunk 3]
```diff
@@ -1,18 +1,44 @@
...
+        mapElement: <div style={{height: '100%'}}/>,
```
Comment (hannahli23) [inline client/src/components/MapContainer.js]: same here, double quotes

File: client/src/components/map/AutocompleteV2.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
+import React from 'react';
```
Comment (hannahli23) [inline client/src/components/map/AutocompleteV2.js:1]: For this entire file, replace single with double quotes

File: client/src/components/map/GmAutocomplete.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+class GmAutocomplete extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            city: "",
+            query: ""
+        }
```
Comment (hannahli23) [inline client/src/components/map/GmAutocomplete.js:9]: Semicolon here apparently and don't forget to do the use double quotes thing

**Review by hannahli23 — CHANGES_REQUESTED** [review]
I reviewed the code and it looked fine, so I ended up just making some codacy comments

**Comment by 9353oasis** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>ℹ</td>
      <td>This pull request includes client dependency changes.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9bb9da69119f7cfe4bd5fb319d57be6c95be43c7
</p>

**Comment by evan10s** [comment]
I'm gonna close this for now and re-open as a wizard PR once that feature is done

**Pull request closed without merging**